addons/crm/models/crm_lead.py

```diff
@@ -353,6 +359,16 @@ def _compute_expected_revenue(self):
         for lead in self:
             lead.expected_revenue = round((lead.planned_revenue or 0.0) * (lead.probability or 0) / 100.0, 2)
 
+    @api.depends('recurring_revenue', 'recurring_plan')
+    def _compute_expected_monthly_revenue(self):
+        for lead in self:
+            lead.expected_monthly_revenue = (lead.recurring_revenue or 0.0) / (lead.recurring_plan.number_of_months or 1)
```

Comment:
I would add round(formula, 2)

Reply:
As @std-odoo pointed out it's a Monetary field, it seems it's already rounded as it should (I remember testing it)
Is there another reason to use the round method ?

Reply:
indeed ! nevermind